PluginCode/src/main/java/smr/cs/ualberta/libcomp/DatabaseAccess.java

```diff
@@ -31,6 +33,7 @@
     private static Session session = null;
     private int userid = 0;
     private String filePath = PathManager.getPluginsPath()+"\\Library_Comparison\\lib";
+    private String feedbackUrllink = "https://smr.cs.ualberta.ca/comparelibraries/api/pluginfeedback/";
```

Comment:
It might be a good idea (perhaps open a new enhancement issue) to move all urls, paths etc to a configuration file. let's say we changed the REST API urls, we shouldn't need to edit or recompile the code.

Reply:
Ok I opened a new issue for this